motr/client.h

```diff
@@ -634,7 +645,7 @@ enum m0_entity_type {
 	 * If this flags is set during entity_create() that means application
 	 * do not support update operation. This flag is not in use yet.
 	 */
-	M0_ENF_NO_RMW =  1 << 1,
+	M0_ENF_NO_RMW = 1 << 1,
```

Comment:
Can you set M0_OOF_NO_RMW if M0_ENF_NO_RMW is true ?

Reply:
@yeshpal-jain-seagate mentioned that we want to skip triggering rmw in case of last IO launch only.
If we applied M0_OOF_NO_RMW if M0_ENF_NO_RMW is set, then it would be applied for all IO launches as in motr clinet we won't know which io launch is last io launch.

If we want to skip rmw for the entire object and not per io launch, then I can simply remove M0_OOF_NO_RMW flag and skip rmw when M0_ENF_NO_RMW flag is set.

Reply:
For Last parity group @andriytk is introducing 00F_LAST https://github.com/Seagate/cortx-motr/pull/1866 , so in this PR continue with either ENF_NO_RMW or OOF_NO_RMW implementations or both can be combined .

Reply:
As we need NO_RMW only in case of the last parity group - maybe we, indeed, don't need this NO_RMW flag at all and use the LAST flag to avoid rmw? This way, we will not create opportunities for the user to corrupt its own data by some misuse of the client API by mistake.